Updated copy of the 2023 volunteer page. (#600)
EuroPython/website: src/content/pages/volunteers.mdx

Being a volunteer means giving your time for free, without expecting anything back, just for the pleasure of being a part of this amazing conference. It's a great way to meet new people, learn new things, and have a lot of fun. You can volunteer for a few hours or for the whole conference, it's up to you.

We will also have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.

# Things to remember

* This is in person volunteering and would require you to be present at the venue.
* While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.
* To enter the conference you need a ticket.
* You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us.

# Perks for volunteers

* Get an awesome EuroPython Volunteer T-Shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-)
* Be part of a great team and make new friends.

> * A special group photo with all volunteers (time will be announced on the #onsite-volunteers)

Reviewer: ```suggestion
* A special group photo with all volunteers (time will be announced on the #onsite-volunteers on discord)
```

We want to call out that this job is located at the registration desk, and you will generally not be able to attend talks for the duration of the task.

## Room Manager

A room manager is responsible for ensuring everything in a room runs smoothly for the entirety of a slot. Some of the activities a room manager is expected to carry out involve:

* Insuring people are seated before a talk, and the room is not overcrowded
* Making sure the session chair and speakers are in the room in time
* Keeping in touch with the AV team to ensure everything is in order
* Keeping the Code of Conduct (CoC) in mind and contact CoC committee members if needed

We want to call out that room managers might not be able to attend or focus on talks for the duration of the task.

## Session Chairing

A session chair plays a key role in introducing talks and ensuring a seamless transition between talks scheduled during a slot. The activities that a session chair is expected to carry out include:

* Helping speakers in preparing their computers for the talks
* Introducing the speakers to the audience
* Signalling time left to the speakers
* Making sure talks start and finish on time

> * Moderating Q&A session and help with passing around microphones and introduce remote attendees.

Reviewer: ```suggestion
* Moderating Q&A sessions and helping with passing around microphones and introducing remote attendees.
```

Author: This sentence needs some more work. Shouldn't "A and B and C" be written as "A, B, and C" ?

Reviewer: Yeah I thought that the sentence has too many and. But I wasn't sure if introducing remote attendees was part of the "helping with". Perhaps it is clearer as "Moderating Q&A sessions, assisting with passing the microphones and introducing remote attendees."?
I was also not sure what "introducing remote attendees" exactly means but I didn't look into the meaning of these tasks much :)

Author: I think you just found a very valid point. (I did not think much either, just C&P from last year and edit where found).
Questions from remote are no longer introduced by the remote person but where read out from discord (if found).
What do you think about
"Moderating Q&A sessions, assisting with passing the microphones and read out questions from remote attendees."
?

Reviewer: sounds good to me - that is if you are sure there will be remote questions as last year :)

Author: Updated as discussed.

> ## Set Up/Tear Down

Reviewer: ```suggestion
## Setup/Teardown
```

* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare etc.)
* Also on the last day (Friday afternoon for the main conference, Sunday for the sprints) to clear up the venue afterward

> * This job involves walking and carrying items, some of which maybe heavy.

Reviewer: ```suggestion
* This job involves walking and carrying items, some of which may be heavy.
```

Runners and Ad-hoc Activities

There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):

* Checking rooms to ensure all the necessary resources are in place
* Helping out with Registration Desk duties, T-Shirt Handout, or Room Manager duties
* Moving chairs, tables, or poster walls
* Arranging transfers between venue and airport, etc. (Rarely)

# How to sign up

We are currently still working out the latest details for the volunteering program. (We are trying to get more perks enabled for your volunteering time).
Until then, you can sign up for volunteering by sending an email to [volunteers@europython.eu](mailto:volunteers@europython.eu) .

> <div class="text-center">

Reviewer: ```suggestion
<div class="text-center mb-8">

```

> <div style={{textAlign: "center"}}>

Reviewer 2: ```suggestion
<div class="text-center">
```

Author: I copied from visa. But looks somehow better to me.

<ButtonLink href="mailto:volunteers@europython.eu">volunteers @ europython.eu</ButtonLink>
</div>

# More information
## Why be a EuroPython Volunteer?

* To help the EuroPython organizers keep the conference affordable for everyone
* Show your EuroPython support to the Python community (by wearing your exclusive T-Shirt)
* Get a huge applause during the closing session of the conference
* Get your name on our EuroPython 2024 Team page to stay there forever.

## How much work is involved?

* There are different tasks with different time required
* Each task is related with the concept of slot (Usually bracketed by the breaks)
* You can sign up to any number of them.

> * We would really appreciate if you can sign up for more than one slot.

Reviewer: ```suggestion
* We would really appreciate it if you can sign up for more than one slot.
```

## What kind of qualifications do I need?

* You should be able to speak some English.
* If you speak additional languages, even better. Please let us know.

> * Have time between two talks? Then why not spend it as an EuroPython Volunteer on the spot?

Reviewer: ```suggestion
* Have time between two talks? Then why not spend it as a EuroPython Volunteer on the spot?
```

Author: Why "a" instead of "an"? For me the sounding is "Uro" which is a vowel and therefore falls into the "an" category. Which rule should I brush up on?

Reviewer: So, the phonetics of "euro" is  /ˈjʊə.rəʊ/. It starts with a /j/ (a bit like "y" in "yellow") which is a consonant sound, thus it should be "a" in this case. If you say "EPS volunteer", it sounds with E ( /iː/), which is a vowel sound, where "an" should be used.

Author: Facepalm. Yes. Thank you.
